### ISSUE TITLE
NPE in org.docx4j.TraversalUtil#getChildrenImpl (6.1.1)

### ISSUE THREAD

**Issue by epihel** [plutext/docx4j#344]
`org.docx4j.TraversalUtil#getChildrenImpl` line 424:
```
artificialList.addAll(handleGraphicData(graphic.getGraphicData()));
```
calls `TraversalUtil#handleGraphicData`, which can return null on line 299:
```
if (wsp.getTxbx()==null) {
	return null;
}
```
If it does, `TraversalUtil#getChildrenImpl` line 424 tries to add `null` to the `ArrayList` `artificialList` and a `NullPointerException` is thrown:
```
java.lang.NullPointerException
	at java.util.ArrayList.addAll(ArrayList.java:577)
	at org.docx4j.TraversalUtil.getChildrenImpl(TraversalUtil.java:424)
	at org.docx4j.TraversalUtil$CallbackImpl.getChildren(TraversalUtil.java:230)
	at org.docx4j.TraversalUtil$CallbackImpl.walkJAXBElements(TraversalUtil.java:120)
	at org.docx4j.TraversalUtil$CallbackImpl.walkJAXBElements(TraversalUtil.java:222)
	at org.docx4j.TraversalUtil$CallbackImpl.walkJAXBElements(TraversalUtil.java:222)
	at org.docx4j.TraversalUtil$CallbackImpl.walkJAXBElements(TraversalUtil.java:222)
	at org.docx4j.TraversalUtil$CallbackImpl.walkJAXBElements(TraversalUtil.java:222)
	at org.docx4j.TraversalUtil$CallbackImpl.walkJAXBElements(TraversalUtil.java:222)
	at org.docx4j.TraversalUtil$CallbackImpl.walkJAXBElements(TraversalUtil.java:222)
	at org.docx4j.TraversalUtil.<init>(TraversalUtil.java:269)
```

**Comment by LennartCockx** [comment]
I'm getting this same error.
Strangely enough this only happens to certain file.

I have a certain docx file which was working properly, then after our client modified the file in word to fix a layout issue, variablereplace started giving this exception.

**Comment by plutext** [comment]
Fix is in docx4j 6.1.2, which has now been released.

**Comment by epihel** [comment]
Confirmed that bug is fixed in 6.1.2. Thanks!
